packages/bpk-component-infinite-scroll/src/withInfiniteScroll-test.flow.js

```diff
@@ -88,10 +88,10 @@ const InfiniteList = withInfiniteScroll(List);
       elementsPerScroll={5}
       initiallyLoadedElements={1}
       loaderIntersectionTrigger="small"
-      onScroll={evt => {}} // eslint-disable-line no-unused-vars
-      onScrollFinished={evt => {}} // eslint-disable-line no-unused-vars
+      onScroll={(evt) => {}} // eslint-disable-line no-unused-vars
```

Comment:
Will prepending an underscore to the var satisfy the lint rule?

Reply:
Are you referring to the `evt` param? I guess we could address this is a separate PR as this ignore has not been added in this PR. 

On the flipside this file will be deleted when we move to TS as its a flow test 😄

Reply:
ah true, ignore